docs/modules/develop/pages/classes/commands.adoc

```diff
@@ -0,0 +1,84 @@
+= Commands
+
+A command is custom class that manages the logic between your input, usually a xref:develop:classes/forms.adoc[form] object, a object and an actor.
+
+Usually, a command is used to create, update or delete a object.
+
+Commands are located in the `app/commands/decidim/<my_plugin>` directory, and named: `<my_command>.rb`.
+
+An example of a command class would:
+
+```ruby
+# frozen_string_literal: true
+
+module Decidim
+  module MyPlugin
+    # A command with the business logic to invite an user to an organization.
+    class CreateMyResource < Decidim::Command
+      # Public: Initializes the command.
+      #
+      # form
+      # resource
```

Comment:
Does it makes sense to have the params as a comment?

Reply:
not in that form. I need to remove it :)